### DISCUSSION TITLE
Wormhole leap year guard

### DISCUSSION THREAD

**Discussion by firecow** [laravel/framework#46338] [General] [answered]
We keep running into issues with developers using `$this->travel(366)->days();`

https://github.com/laravel/framework/blob/10.x/src/Illuminate/Foundation/Testing/Wormhole.php#L142
Would it make sense to add guard that throws exception here, if `$this->value >= 365`?

**Comment by rojtjo** [comment, marked as answer]
I wonder what kind of issues you are running into here?

Throwing an exception wouldn't make sense as there might be legitimate use case that requires them to travel for more than 365 days.

**Reply by firecow** [reply]
Yeah, you're right... 👍

We just keep running into leap year issues.
But those are just stupid mistakes, and should be treated as such.
